CodeEdit/AppDelegate.swift

```diff
@@ -144,10 +144,14 @@ class AppDelegate: NSObject, NSApplicationDelegate, ObservableObject {
                         }
                     }
 
-                } else {
-                    CodeEditDocumentController.shared.openDocument { _, _ in
+                }
+                else {
```

Comment:
move the `else` statement to the previous line:

```swift
if ... { 
  ...
} else {
  …
}
```

having it on a new line fails in `SwiftLint`